tricicl/research/mathieu/benchmark_split_mnist.py

```diff
@@ -34,7 +35,8 @@ def evaluate(name: str, plugins: list[StrategyPlugin]):
         evaluator=EvaluationPlugin(
             accuracy_metrics(experience=True, stream=True),
             StreamConfusionMatrix(
-                num_classes=perm_mnist.n_classes, image_creator=SortedCMImageCreator(perm_mnist.classes_order),
+                num_classes=perm_mnist.n_classes,
+                image_creator=SortedCMImageCreator(perm_mnist.classes_order),
```

Comment:
What caused this ? The line was 111 chars long

Reply:
Yeah I am going with 120 char limit so the fact that it is 111 char long could not have been the issue. I believe it was a styling issue? I ran black and it kept breaking it up :thinking:  I am also not so successful with the documentation of black, they say:
>If the contents of the matching brackets pair are comma-separated (like an argument list, or a dict literal, and so on) then Black will first try to keep them on the same line with the matching brackets. If that doesn’t work, it will put all of them in separate lines.

But this is not detailed enough to see what is the exact condition(s) it follows to **_put all of them in separate lines_**. Would you happen to have any hypothesis?

Reply:
I don't understand, black undo your breaking on my side 😅 
Anyway not very important, but we might have some unwanted changes because of that later on

Reply:
This is annoying I will try to see if I can find the issue on my end, just to double check (it might not be relevant) is your black running on python 3.7 or 3.9 version?

Reply:
I was using the default auto-detection, but I just tried with py37 and it is the same result

what is your black version ? mine is `19.10b0`

Reply:
I am running on black version `20.8b1`

Reply:
That was the issue then, black20 and black19 don't behave the same way in that case... I'll use your version, and we shouldn't have this issue anymore, except for parts I already commited

Reply:
(What's funny though is that if you remove the coma at the end of the last line, black20 put everything on the same line, as 19 was doing)